packages/office-ui-fabric-react/src/components/ColorPicker/ColorPicker.tsx

```diff
@@ -92,48 +92,58 @@ export class ColorPicker extends BaseComponent<IColorPickerProps, IColorPickerSt
             <tbody>
               <tr>
                 <td>
-                  <TextField
+                  <DefaultTextField
```

Comment:
Um. Why are you renaming everything that was TextField to DefaultTextField? That feels counterintuitive.